resources/build/Dockerfile

```diff
@@ -0,0 +1,45 @@
+#
```

Comment:
[fix] SPDX license identifier and copyright.

Reply:
[ac60b1a](https://github.com/OpenAssetIO/OpenAssetIO/pull/735/commits/ac60b1a665d29214d4d6afb096ae9c83ab08b88a)